apps/src/dev/app.html.ejs

```diff
@@ -12,10 +12,6 @@
   <link rel="stylesheet" type="text/css" href="../css/common.css">
   <link rel="stylesheet" type="text/css" href="../css/<%= app %>.css">
 
-<% if (app === "applab") { %>
-  <link rel="stylesheet" type="test/css" href="../css/colpick.css">
-<% } %>
```

Comment:
Looks unlikely to be used there either, since that is only for applab. Will remove the css file.